ci/infra/openstack/generate-cloud-provider-conf

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+umask 0077
```

Comment:
This is because default Linux file mask is coded on 4 digits, first digit is defining `special mode` for
```
4 - setuid
2 - setgid
1 - sticky-bit
0 - no special mode for file and it is ignored
```

Reply:
I have changed it